ui/store/actions.ts

```diff
@@ -4851,9 +4852,44 @@ export function getGasFeeTimeEstimate(
   ]);
 }
 
-export async function closeNotificationPopup() {
+export async function attemptCloseNotificationPopup() {
+  // Check if the current window is NOT a popup - if confirmed, we should not close it
+  try {
+    const currentWindow = await browser.windows.getCurrent();
+    if (currentWindow.type && currentWindow.type !== 'popup') {
+      console.warn(
+        `Not safe to close a window that is not a popup: It is of type: ${currentWindow.type}`,
+      );
+      // We've confirmed this is not a popup window, so it's not safe to close.
+      return;
+    }
+  } catch (error) {
+    // Error occurred while checking window type - we cannot confirm it's not a popup, so continue
```

Comment:
For the sake of making debugging easier, lets not completely suppress this error. We can use `console.debug` if it's not something we want logged in production, or `console.log` if it's something we might want to see in logs from support tickets.

I'd recommend `console.warn`, so that if a window stays open when it shouldn't, we'll see this hint in the logs as to why. Even if it's encountered on a production build by an end-user.

Reply:
Er... sorry, I misunderstood at first. We're assuming it _is_ a popup if this fails.

Still seems potentially useful to log here, and not disruptive give that nobody would see it. But maybe less helpful than I originally thought.

Reply:
Added a warn.